MapboxNavigation/NavigationMapView.swift

```diff
@@ -1035,6 +1043,30 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         return MGLShapeCollectionFeature(shapes: [mainRoute])
     }
     
+    /**
+     Returns an array of `MapboxStreetsRoadClass` objects for specific leg.
+     
+     - parameter leg: Leg of the route.
+     - returns: An array of `MapboxStreetsRoadClass` for specific `RouteLeg`. `MapboxStreetsRoadClass` will be set to `nil` if it's not present in `Intersection`.
+     */
+    func streetsRoadClasses(_ leg: RouteLeg) -> [MapboxStreetsRoadClass?] {
```

Comment:
The `_` label isn’t warranted here because `leg` isn’t a road class. Actually, it looks like this method doesn’t use `self` anywhere, so it would be more natural to transform this method into a computed property of an extension of RouteLeg.